src/Microsoft.TestPlatform.VsTestConsole.TranslationLayer/VsTestConsoleProcessManager.cs

```diff
@@ -121,7 +121,6 @@ public void StartProcess(ConsoleParameters consoleParameters)
 
         EqtTrace.Verbose("VsTestCommandLineWrapper: Process Start Info {0} {1}", info.FileName, info.Arguments);
 
-#if NETFRAMEWORK
         if (consoleParameters.EnvironmentVariables != null)
         {
             info.EnvironmentVariables.Clear();
```

Comment:
Unlike the change for 17.3, this keeps the behavior of deleting all variables if you provide any. It just enables the api for .net core.